firmware/util/containers/table_helper.h

```diff
@@ -69,28 +69,28 @@ class Map3D : public ValueProvider3D {
 		m_values = reinterpret_cast<TValue (*)[TRowNum][TColNum]>(&table);
```

Comment:
Do you want to update this function?  Or doesn't matter since it only affects values, not keys?